### ISSUE TITLE
ERROR

### ISSUE THREAD

**Issue by jsrider** [justinmc/react-audio-player#21]
```
ERROR in ./src/index.jsx

/Users/weishao/Aliyun/react/react-audio-player/src/index.jsx
  127:32  error  Missing trailing comma  comma-dangle

✖ 1 problem (1 error, 0 warnings)

webpack: Failed to compile.
```

**Comment by justinmc** [comment]
Looks like a comma was indeed missed on that line.  I'll push a quick fix for this tonight and publish a new patch version.  Thanks for pointing this out!

**Comment by justinmc** [comment]
Confession, I forgot about this and left on a week long trip to Ecuador with no laptop. I will get this asap when I'm back, sorry about that.

**Comment by justinmc** [comment]
Done and released with version 0.3.3.  Sorry for the delay!
